common/Strings/en-us/Resources.resw

```diff
@@ -205,12 +205,32 @@
     <value>Uptime:</value>
     <comment>Title for the uptime property of a compute system. (The amount of time its been running)</comment>
   </data>
+  <data name="HyperVAdminAddUser" xml:space="preserve">
+    <value>Add user to {0} administrators group</value>

```

Comment:
Should Administrators be capitalized? Also, these strings are inconsistent with "administrators" and "administrator".